active/events-support.md

```diff
@@ -0,0 +1,55 @@
+# Events Support Working Group
+
+## Scope of responsibilities
+
+The group supports organizers of Django-related events globally, including DjangoCon conferences. The group acts on behalf of the DSF Board to:
+
+- Be the point of contact for organizers to ask questions and seek guidance.
+- Proactively provide support to event organizers.
+- Debrief with organizers after events.
+- Maintain shared resources to facilitate continuity and knowledge transfer between events.
+- Actively promote diversity and inclusion in event support practices, encouraging organizers to create inclusive events that welcome and represent diverse communities.
+
+For DjangoCon events, the working group will:
+
+- Liaise with DEFNA (Django Events Foundation North America) who oversee DjangoCon US.
+- Manage and execute the Call for Organizers process for DjangoCon Europe, providing recommendations to the DSF board for final approval.
+- Assist DjangoCon Europe organizers with event planning, logistics, and execution.
+- Work with DjangoCon Africa organizers to make this event sustainable.
+- Work with the Social Media Working Group to promote events
+
+## Initial membership
+
+The group has a small initial membership, with the goal of expanding to 7-10 members able to represent
+
+Current members:
+
+- Chair: Thibaud Colas
+- Co-Chair: David Vaz
+- Board Liaison: Thibaud Colas
+- Other members:
+  - You?
```

Comment:
```suggestion
  . Benjamin Balder Bach
  - You?
```

Reply:
I'm nominating myself with two skills:

* Being a member of the former DjangoCon Europe Support Group
* Being an organizer of DjangoCon Europe 2019

I'd like to continue to help foster happy, creative, sustainable Django-related events. Both big, but especially also small :)